alf/experience_replayers/replay_buffer.py

```diff
@@ -18,9 +18,17 @@
 import torch.nn as nn
 
 import alf
+from alf.data_structures import namedtuple
 from alf.nest.utils import convert_device
+from alf.utils.common import warning_once
 from alf.utils.data_buffer import atomic, RingBuffer
 
+from .segment_tree import SumSegmentTree, MaxSegmentTree, MinSegmentTree
+
+BatchInfo = namedtuple(
+    "BatchInfo", ["env_ids", "positions", "importance_weights"],
+    default_value=())
+
 
 class ReplayBuffer(RingBuffer):
     """Replay buffer with RingBuffer as implementation.
```

Comment:
Add comment on what behavior to expect for this replay buffer if `prioritized_sampling=True` for mini_batch_len=1 and >1